tests/test_esm_runscripts/test_filedicts.py

```diff
@@ -412,6 +412,32 @@ def test_ln_raises_exception_when_target_path_does_not_exist(simulation_file, fs
 
 # ========== end of ln() tests ==========
 
+def test_check_file_info_is_complete():
```

Comment:
To keep the test case short, the fixture can be used. I already provided in the first commits.

Reply:
This also is a global style choice we need to agree upon.

Reply:
This will be unify on a later PR after we agree on a common strategy